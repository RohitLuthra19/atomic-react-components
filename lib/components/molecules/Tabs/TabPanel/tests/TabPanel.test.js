import { shallow } from 'enzyme';

import TabPanel from '../index';

describe('<TabPanel />', () => {
  let TabPanelComponent = '';
  beforeEach(() => {
    TabPanelComponent = shallow(<TabPanel>Test</TabPanel>);
  });

  test('should render correctly', () => {
    expect(TabPanelComponent).toMatchSnapshot();
  });
});
