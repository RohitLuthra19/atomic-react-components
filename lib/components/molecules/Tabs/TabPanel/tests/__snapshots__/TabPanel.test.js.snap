// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabPanel /> should render correctly 1`] = `
<TabPanel
  className="TabPanel-f4hm5l-0 jlQTVu"
  inheritedStyles=""
>
  Test
</TabPanel>
`;
